esp/src/src-react/components/controls/Grid.tsx

```diff
@@ -308,30 +298,8 @@ const FluentStoreGrid: React.FunctionComponent<FluentStoreGridProps> = ({
         columnWidths.set(column.key, newWidth);
     }, [columnWidths]);
 
-    /*  Monitor Scroll Events (hack)
-
-        Essentially we are setting the scrollElement of the DetailsList to the div that contains the DetailsList (rather than a scrollable pane host).
-        See:  https://github.com/microsoft/fluentui/blob/55d3a31042e8972ea373841bef616c68e6ab69f9/packages/react/src/components/List/List.tsx#L355-L369
-
-        Note: Not sure if `_onScroll` call is needed, but excluding for now as it seems to work without it and is more performant.
-    */
-    const id = useId("fluent-store-grid-");
-    const detailListComponent = React.useRef<IDetailsListEx | null>(null);
-    const [detailListElement, setDetailListElement] = React.useState<HTMLDivElement | null>(null);
-    useMount(() => {
-        const detailListElement = document.querySelector<HTMLDivElement>(`#${id} .ms-DetailsList`);
-        setDetailListElement(detailListElement);
-        if (detailListComponent.current?._list?.current) {
-            detailListComponent.current._list.current._scrollElement = detailListElement;
-        }
-    });
-    useOnEvent(detailListElement, "scroll", () => {
-        detailListComponent.current?._list?.current?._onAsyncScrollDebounced();
-    });
-
-    return <div id={id} data-is-scrollable={"true"} style={{ overflow: "auto" }}>
+    return <div data-is-scrollable="true" style={{ overflow: "auto" }}>
```

Comment:
@GordonSmith Could you change this to `style={{ overflow: "auto", height: "100%" }}`? While I was testing this, I noticed there's a separate issue that I think is related to these changes, where the pagination on the Grids are getting the top 10 or 20 pixels cut off

Reply:
Done

Reply:
If you can sanity check that I will merge in the morning...